services/webnn/ort/graph_builder_ort.cc

```diff
@@ -337,6 +337,40 @@ GraphBuilderOrt::CreateOrReshapeBias(const std::optional<uint32_t>& bias_id,
   return bias;
 }
 
+std::string GraphBuilderOrt::ConvertRznToZrn(std::string_view weight_or_bias) {
+  uint32_t num_gates = 3;
+
+  // Use Split operator to split the weight/bias into num_gates slices.
+  std::string r_gate = GenerateNextOperandName();
+  std::string z_gate = GenerateNextOperandName();
+  std::string n_gate = GenerateNextOperandName();
+  std::vector<ScopedOrtOpAttr> split_attrs;
+  split_attrs.reserve(2);
+  split_attrs.push_back(
+      model_editor_.CreateAttribute("axis", static_cast<int64_t>(1)));
+  split_attrs.push_back(model_editor_.CreateAttribute(
+      "num_outputs", static_cast<int64_t>(num_gates)));
```

Comment:
Nit: just use `3`, the "num_outputs" is informative
```suggestion
      "num_outputs", static_cast<int64_t>(3)));
```